java/sage/Wizard.java

```diff
@@ -15,66 +15,34 @@
  */
 package sage;
 
-import static sage.SageConstants.LITE;
-
 import org.apache.lucene.analysis.Analyzer;
 import org.apache.lucene.document.Document;
 import org.apache.lucene.document.Field;
-import org.apache.lucene.index.CorruptIndexException;
-import org.apache.lucene.index.IndexCommit;
-import org.apache.lucene.index.IndexReader;
-import org.apache.lucene.index.IndexWriter;
-import org.apache.lucene.index.IndexWriterConfig;
+import org.apache.lucene.index.*;
```

Comment:
The comments are in markdown, so you need to escape certain characters. \* It bugs me too sometimes.

I do strive to be a good contributor to this open source project and I have tried to emulate some of the style I've seen even when it's not my preference. I will fix the imports.